src/net/sourceforge/kolmafia/StaticEntity.java

```diff
@@ -81,17 +81,33 @@ public static final String getVersion() {
     if (StaticEntity.cachedVersionName == null) {
       StringBuilder versionName =
           new StringBuilder(PRODUCT_NAME).append(" r").append(StaticEntity.getRevision());
+
+      if (isCodeModified()) {
+
+        versionName.append("-M");
+      }
+
       StaticEntity.cachedVersionName = versionName.toString();
     }
 
     return StaticEntity.cachedVersionName;
   }
 
+  public static final boolean isCodeModified() {
```

Comment:
Does this need to be public? (Do we want it to be for some future use case?)

Reply:
nope, maybe, but it can change then.